### DISCUSSION TITLE
SignalR Blazor

### DISCUSSION THREAD

**Discussion by gerardocrr** [dotnet/aspnetcore#55021] [Q&A]
I have this project, I literally copied and pasted the code, in visual studio it works correctly, but when I upload it to my iis it doesn't work, does anyone know why?
https://learn.microsoft.com/en-us/aspnet/core/blazor/tutorials/signalr-blazor?view=aspnetcore-8.0&tabs=visual-studio

https://github.com/dotnet/aspnetcore/assets/89473397/b9ca1e4c-42c9-46f4-8959-135738c7f819

**Comment by davidfowl** [comment]
"Doesn't work" is a bit vague. What sort of errors did you get? Can you share machine details? IIS setup, what you have installed? etc etc

**Reply by davidfowl** [reply]
Is this windows 11 home edition?

**Reply by gerardocrr** [reply]
Is Windows 11 Home Single Language

I have IIS 10.0.22
.NET8

**Reply by davidfowl** [reply]
That hanging behavior looks like the fun limitation windows client has around 10 concurrent websocket connections. Windows server doesn't have this limitation. 

I'm not 100% sure that's the problem though. I bet it'll work in IIS Express or on a windows server. Is there any reason you're using iis?

**Reply by gerardocrr** [reply]
In visual studio it works fine with iis express, I was testing iis to deploy a local project that I want to do with signalR.
